workflow-templates/lint-info-xml.yml

```diff
@@ -7,7 +7,6 @@ name: Lint
 
 on:
   pull_request:
-  push:
     branches:
```

Comment:
I am aware that the branches moved up from push to pull_request. this just expresses that only PRs aganist main, master and stable* should be built, which is reasonable.